tests/.editorconfig

```diff
@@ -0,0 +1,45 @@
+# EditorConfig is awesome:http://EditorConfig.org
+
+[*.cs]
+# We allow usage of "var" inside tests as it reduces churn as we remove/rename types
+csharp_style_var_for_built_in_types = true:none
+csharp_style_var_elsewhere = true:none
+
+# Dotnet code analysis settings:
+[*.{cs,vb}]
+# Microsoft.CodeQuality.Analyzers
+# For tests, the ConfigureAwait(true) is good enough. Either they are already running on a thread pool
```

Comment:
This comment seems out of date and applicable to none of these CAXXX rules - was it in a different location before the ruleset -> editorconfig?

Reply:
We don't enforce ConfigureAwait anywhere - so the comment is kindof meaningless now.